Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM ubuntu:20.04
 RUN apt-get update && apt-get upgrade --yes && apt-get install git python3.8 python3-pip --yes
 COPY . .
+RUN pip install typing-extensions==3.10.0.2
```

Comment:
Add a comment why this is necessary

Reply:
yeah or a TODO to remove once this stuff we are able to us typing-extensions>=4.0.0

Reply:
ok cool I added a note and a todo to remove it

Reply:
I updated the TODO so that it is more descriptive and to figure out a better solution.